### PR TITLE
[Term Entry] Dart Type Conversion: .toStringAsPrecision()

### PR DESCRIPTION
**Pull request by tomasbaublys**
### Description

Added the .toStringAsPrecision() term for Dart.
Closes issue https://github.com/Codecademy/docs/issues/4567

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Mobile Development'
+Tags:
+  - 'Dart'
+  - 'Data Types'
+  - 'Methods'
+  - 'Numbers'
+  - 'Strings'
```
Comment (mohitO12-dev) [inline content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:13]: PERFECT

File: content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+Title: '.toStringAsPrecision()'
+Description: 'Converts this number to a double and returns a string representation of that value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Mobile Development'
```
Comment (mohitO12-dev) [inline content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:7]: PERFECT

File: content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+Title: '.toStringAsPrecision()'
+Description: 'Converts this number to a double and returns a string representation of that value.'
```
Comment (mohitO12-dev) [inline content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:3]: PERFECT

File: content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+1234567.toStringAsPrecision(9); 
+```
+
+The above code produces the following outputs:
+
+```shell
+Output = 1.0
+Output = 1.00e+15
+Output = 1234567.00
+```
```
Comment (mohitO12-dev) [inline content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:44]: PERFECT DART CODE AND OUTPUT.

Comment (mohitO12-dev) [reply content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:44]: @tomasbaublys please sign the CLA document.

Comment (mohitO12-dev) [reply content/dart/concepts/type-conversion/terms/toStringAsPrecision/toStringAsPrecision.md:44]: remove the last entry or just connect with me on teams or slack @tomasbaublys ❌ Tomas Baublys

**Review by mohitO12-dev — APPROVED** [review]
APPROVED

**Comment by mamtawardhani** [comment]
> ✅ tomasbaublys
> ✅ mohitO12-dev
> ❌ Tomas Baublys

Hey, @tomasbaublys! Seems like you have used two GitHub accounts for this entry, I would request you to sign the CLA with the other GitHub account as well. Thank you!

**Comment by tomasbaublys** [comment]
> > ✅ tomasbaublys
> > ✅ mohitO12-dev
> > ❌ Tomas Baublys
> 
> Hey, @tomasbaublys! Seems like you have used two GitHub accounts for this entry, I would request you to sign the CLA with the other GitHub account as well. Thank you!

@mamtawardhani Yes, but I can't understand how to sign the lats one - ❌ Tomas Baublys

**Comment by mamtawardhani** [comment]
> > > ✅ tomasbaublys
> > > ✅ mohitO12-dev
> > > ❌ Tomas Baublys
> > 
> > 
> > Hey, @tomasbaublys! Seems like you have used two GitHub accounts for this entry, I would request you to sign the CLA with the other GitHub account as well. Thank you!
> 
> @mamtawardhani Yes, but I can't understand how to sign the lats one - ❌ Tomas Baublys

You will have to sign in to the other GitHub ID as well it seems and then sign the CLA, can that be done?

**Comment by tomasbaublys** [comment]
> > > > ✅ tomasbaublys
> > > > ✅ mohitO12-dev
> > > > ❌ Tomas Baublys
> > > 
> > > 
> > > Hey, @tomasbaublys! Seems like you have used two GitHub accounts for this entry, I would request you to sign the CLA with the other GitHub account as well. Thank you!
> > 
> > 
> > @mamtawardhani Yes, but I can't understand how to sign the lats one - ❌ Tomas Baublys
> 
> You will have to sign in to the other GitHub ID as well it seems and then sign the CLA, can that be done?

But I have only one account ✅ tomasbaublys that's why I can't sign second...

**Comment by mamtawardhani** [comment]
@tomasbaublys 
Could you please check if your Git account is configured for this PR in your local terminal by using the following commands in git bash:

```
git config user.email
git config [user.name](http://user.name/)
If these are not configured, please set them using the commands below:
git config --global user.email <[your_correct_email@example.com](mailto:your_correct_email@example.com)>
git config --global [user.name](http://user.name/) <your_correct_username>
```

**Comment by tomasbaublys** [comment]
> @tomasbaublys Could you please check if your Git account is configured for this PR in your local terminal by using the following commands in git bash:
> 
> ```
> git config user.email
> git config [user.name](http://user.name/)
> 
> If these are not configured, please set them using the commands below:
> git config --global user.email <[your_correct_email@example.com](mailto:your_correct_email@example.com)>
> git config --global [user.name](http://user.name/) <your_correct_username>
> ```

@mamtawardhani My git account were configured correctly, here you can see output:

```
user.email=t.baublys@icloud.com
user.name=tomasbaublys
```

**Comment by mamtawardhani** [comment]
> > @tomasbaublys Could you please check if your Git account is configured for this PR in your local terminal by using the following commands in git bash:
> > ```
> > git config user.email
> > git config [user.name](http://user.name/)
> > 
> > If these are not configured, please set them using the commands below:
> > git config --global user.email <[your_correct_email@example.com](mailto:your_correct_email@example.com)>
> > git config --global [user.name](http://user.name/) <your_correct_username>
> > ```
> 
> @mamtawardhani My git account were configured correctly, here you can see output:
> 
> ```
> user.email=t.baublys@icloud.com
> user.name=tomasbaublys
> ```

Oh, okay! Then @tomasbaublys, I would suggest you to close this PR and raise a new PR for this entry, that would be a better idea.

**Comment by mamtawardhani** [comment]
Hey @tomasbaublys, I wanted to followup on this PR as to when will you be submitting a PR for this issue. Please let me know. Thanks! 😃

**Comment by tomasbaublys** [comment]
Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?

**Comment by mamtawardhani** [comment]
> Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?

Yes you can, and also before you start it on a local repo, make sure your git is configured to your GitHub account. 
You can refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) and look through the reference videos given in the steps. Let me know if you need any help, thanks! 😄

**Comment by tomasbaublys** [comment]
> > Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?
> 
> Yes you can @tomasbaublys , and also before you start it on a local repo, make sure your git is configured to your GitHub account. You can refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) and look through the reference videos given in the steps. Let me know if you need any help, thanks! 😄

@mamtawardhani I did it again :)

**Comment by mamtawardhani** [comment]
> > > Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?
> > 
> > 
> > Yes you can @tomasbaublys , and also before you start it on a local repo, make sure your git is configured to your GitHub account. You can refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) and look through the reference videos given in the steps. Let me know if you need any help, thanks! 😄
> 
> @mamtawardhani I did it again :)



> > > Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?
> > 
> > 
> > Yes you can @tomasbaublys , and also before you start it on a local repo, make sure your git is configured to your GitHub account. You can refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) and look through the reference videos given in the steps. Let me know if you need any help, thanks! 😄
> 
> @mamtawardhani I did it again :)



> > > Hi @mamtawardhani I'm a little bit confused. Should I start all from beginning? Maybe better start new repo locally?
> > 
> > 
> > Yes you can @tomasbaublys , and also before you start it on a local repo, make sure your git is configured to your GitHub account. You can refer to the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide) and look through the reference videos given in the steps. Let me know if you need any help, thanks! 😄
> 
> @mamtawardhani I did it again :)

Thank you @tomasbaublys! Will get it reviewed asap 😄

**Pull request closed without merging**